app/src/main/java/org/greenstand/android/TreeTracker/models/User.kt

```diff
@@ -39,7 +38,17 @@ class User(
 
     var lastCheckInTimeInSeconds: Long?
         get() = preferences.getLong(LAST_CHECK_IN_TIME_IN_KEY)
-        set(value) = preferences.edit().putLong(LAST_CHECK_IN_TIME_IN_KEY, value ?: -1).apply()
+        set(value) = preferences
+            .edit().putLong(LAST_CHECK_IN_TIME_IN_KEY, value ?: -1).apply()
+
+    var absoluteStepCount: Int?
```

Comment:
I am assuming the collected step count is relevant for a specific user session when the planter is capturing trees. Hence adding these in the User model so that they all are removed when user switches or session ends.

Reply:
Yes, that can be done. But isn't `Session` inherently `User` associated?

Reply:
They are associated, but they don't have to be in the same class. When I think of checking my current step count, i'd look for a step count class, not the user class.